app/src/main/java/cm/aptoide/pt/presenter/LoginSignUpCredentialsPresenter.java

```diff
@@ -71,6 +72,26 @@ public LoginSignUpCredentialsPresenter(LoginSignUpCredentialsView view,
     handleTogglePasswordVisibility();
   }
 
+  private void handleClickOnTermsAndConditions() {
+    view.getLifecycle()
+        .filter(event -> event.equals(View.LifecycleEvent.CREATE))
+        .flatMap(__ -> view.termsAndConditionsClickEvent())
+        .doOnNext(__ -> accountNavigator.navigateToTermsAndConditions())
+        .compose(view.bindUntilEvent(View.LifecycleEvent.DESTROY))
+        .subscribe(__ -> {
+        }, err -> crashReport.log(err));
+  }
+
+  private void handleClickOnPrivacyPolicy() {
+    view.getLifecycle()
+        .filter(event -> event.equals(View.LifecycleEvent.CREATE))
+        .flatMap(__ -> view.privacyPolicyClickEvent())
+        .doOnNext(__ -> accountNavigator.navigateToPrivacyPolicy())
```

Comment:
After the flatMap of the click it should have the observeOn on MainThread and retry()

Reply:
fixed